packages/flutter_tools/templates/app/integration_test/app_test.dart.tmpl

```diff
@@ -0,0 +1,49 @@
+// This is a basic Flutter integration test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import 'package:{{projectName}}/main.dart' as app;
+
+void main() {
+  {{! TODO(dnfield): remove/update this once https://github.com/flutter/flutter/pull/70075 lands }}
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Counter increments smoke test', (WidgetTester tester) async {
+    // Build our app and trigger a frame.
+    app.main();
```

Comment:
I'm a bit curious about the best practice here. Is this better than calling `app.main`, or instead importing the root widget and using `tester.pumpWidget`?

Reply:
`app.main` will be the same regardless of what the root widget is. Whereas if you refactor your app to add a new root widget, you'd have to update the integration tests.